### ISSUE TITLE
[Rough Draft] Course [COURSE NAME]: Project Thread + Brainstorm

### ISSUE THREAD

**Issue by wparasae** [WordPress/Learn#828]
### Designing a Course: A Process Checklist From Start to Finish 
**(A Checklist and Brainstorm Template)**

**Use this template if you would like to be a "Course Lead Designer" - A Course Lead Designer is someone who takes an approved course topic (one that has been vetted by a training team member). They work with a faculty member buddy to brainstorm, outline, organize, and help wrangle course content. 

**Their role is to see the course grow from start to finish with the help of a faculty buddy.** 

Before using this template, this course topic should have been approved by the training team; please ask for help in the Slack #training channel if you have questions about this checklist. We are happy to answer!**

**At this point in the process, someone should have...**
- [x] Submitted a course topic proposal (form TBD) _Note: You do not have to be the person who submitted a course topic proposal to be the Course Lead Designer _
- [x] Course Proposal has been reviewed by a Training team faculty member and approved.

If both of those things are true for the course topic you'd like to help with, please continue with the next steps!

**WHAT SHOULD YOU DO FIRST?**
- [ ] I have assigned myself to this course template. _Click on 'Assign Yourself' to assign yourself as the Course Lead Designer_. 

<img width="309" alt="Screen Shot 2022-07-20 at 2 50 54 PM" src="https://user-images.githubusercontent.com/101278360/180059983-1335e697-4326-4c7f-866f-429dcac11a3f.png">

**WHAT SHOULD YOU DO SECOND? COMPLETE THE COURSE BRAINSTORM**
- [ ] COURSE LEAD DESIGNER: Complete the course brainstorm (you can find the questions below) by answering as many of the questions as you can. **_There are no right or wrong answers here, this is just to get your ideas down on paper._** 
- [ ] Submit this as a new issue; please label it with **INSERT PROPER LABELS HERE NOT SURE WHAT THESE ARE YET AND ALSO PROVIDE SCREENSHOTS**
- [ ] Write a message in  the#training channel that you have completed a course brainstorm. A faculty member will assign a buddy to you, and they will reach out to schedule a time to chat in the next week or two.

**STEP THREE: OUTLINE THE COURSE WITH YOUR FACULTY BUDDY**
- [ ] COURSE LEAD DESIGNER: Meet with Course Faculty Buddy to "Chunk" your course into Modules & Lessons - _Your buddy will help you write lesson objectives and chat with you about your awesome course idea!_
- [ ] Review course outline with buddy _**(Should this be done with the Training team, a faculty member, or an SME? Will slow down the process, but might be important for first-time contributors)**_

**STEP FOUR: GATHER AND CREATE RESOURCES**
- [ ] ANY CONTRIBUTOR: Find related resources on Learn.WordPress.org that already exist, link to each of them in this example -- this step can be done with your buddy or independently.
- [ ] ANY CONTRIBUTOR: Find lesson plans that already exist and link them to this course -- this step can be done with your buddy or independently.
- [ ] COURSE LEAD DESIGNER: Create new required lesson plans and add links to them to this this Github issue -- this step can be done with your buddy during a call or async, or you can complete this independently.

**STEP FIVE: START DRAFTING LESSON PLANS IN LEARN (Probably using Sensei)**
- [ ] FACULTY BUDDY: Create course frame in Sensei **(Need to figure out how Lesson Plans / Courses / Content will Merge TBD)**
- [ ] COURSE LEAD DESIGNER and BUDDY: Sensei / Learn Walk-Around (to get familiar with certain tools and how it works on Learn, specifically) **Resource Needed: How to set up courses in Sensei. Sensei Toolbox course perhaps, or a buddy session with an instructional designer**
- [ ] ANY CONTRIBUTOR: Write lesson plans (Google docs or in Sensei frame?) (Or in the lesson plan format, unclear where this is going at the moment...)
- [ ] As each lesson plan is created, it's important to write comments on a Github thread with updates and to check in with the #training team during weekly calls. _I would also recommend collaborating directly with the COURSE LEAD DESIGNER before starting a lesson plan._

**WORKSHOP CREATION** (Optional)
- [ ] ANY CONTRIBUTOR / COURSE LEAD DESIGNER: Identify workshops and create as needed or submit requests for workshops to be created with the training team

**### WHEN ALL THE LESSON PLANS HAVE BEEN DRAFTED (FIRST DRAFTS), WHAT SHOULD YOU DO NEXT?** 
**REVIEW AND MAKE CHANGES TO DRAFTED LESSON PLANS**
- [ ] ANY CONTRIBUTOR: As each lesson plan is created, submit it to the Training team to be reviewed.
- [ ] ANY CONTRIBUTOR: Review submitted lesson plan and suggest changes; 
- [ ] COURSE LEAD DESIGNER should also review and modify lesson plans as they are submitted.
- [ ] COURSE LEAD DESIGNER: Check to make sure all lesson plans have been created, reviewed, and revised in Sensei.

**FINAL COURSE REVIEWS**
- [ ] FACULTY: Reach out to Marketing about Promotions with estimated timeline for final completion.
- [ ] ANY CONTRIBUTOR: Content & Accessibility Review
- [ ] ANY CONTRIBUTOR: Copy Editing 
- [ ] COURSE LEAD DESIGNER + FACULTY BUDDY: Publish that course!
- [ ] ANY CONTRIBUTOR: Celebrate!

### **Course Template Brainstorm: **THE TITLE OF YOUR COURSE***
Course Lead Designer: [Write your name and make sure you have assigned yourself to this issue.]

Brainstorm Questions: You can either complete this brainstorm with the buddy during a call, or you can answer these questions (no wrong answers!) ahead of the scheduled call. The choice is yours as Course Lead Designer.

**1. What is the problem that this course is trying to solve? Describe it.**

**2. Course OBJECTIVE: What is a PROJECT someone should be able to complete if they take this course?**
"By the end of this course, a learner should be able to... (list 1 or 2 project ideas)

Example: __Intermediate users and theme Figma designers will be able to..._
-Build a fully functional, custom WordPress theme with very little coding (theme.json) required._

**3. Who is the audience for this course?

Level
- [ ] Beginners
- [ ] Intermediate
- [ ] Advanced

Type of User
* [ ] Users / Content Writers
* [ ] Designers
* [ ] Developers
* [ ] Speakers
* [ ] Organizers
* [ ] Kids

**_How will this course address their need?_**

**4. _Describe_ the Audience: What should this audience already know and be able to do comfortably? Bonus: What MIGHT this audience already know?**

_Examples: This audience (intermediate users and Figma designers) already knows how to...
-Navigate the WordPress dashboard comfortably
-How to install and update themes
-How to identify a classic theme vs. a block theme
-Use the site editor (including list view, using blocks, using patterns, etc.)_ 

**5. Look at your overall course objective. Brainstorm (no right or wrong answers here):** As you consider the project you're hoping people will be able to do , ask yourself:

- What are the things your audience doesn't know or know how to do related to this topic yet? (List whatever you can think of below -- there are no wrong answers)

- Or, conversely, what are some questions they might ask about your topic?** (List questions your audience might have about this course topic below -- there are no wrong answers)

**6. What resources already exist?** Check learn.wordpress.org for tutorials, lesson plans, courses, and former social learning spaces that already exist. Link to them below. (This can be done with your buddy, by another contributor, or at a later point).

**Other Considerations: (Optional)
- What vocabulary will this audience need to learn? Write a list of terms and definitions as a brainstorm.
- What information do you have? Conversely, what information do you NOT have yet that you need in order to complete this course? _We can work to pair you with a subject matter expert or reach out for other contributor help if needed._
**

**Next steps:** Please reach out in training to let the team know you have completed this brainstorm. You will be assigned a buddy in your timezone who will help you with "chunking" the course and framing it into individual lessons and modules.

**Comment by wparasae** [comment]
**Future Course Template Needs (writing here):**
Course Template (Rough Draft In Progress)
Course Lesson Plan Template (Not Started) - Complete with proper labels.

Note to Self: Creating a Bullet Outline + Brief Description of Every Lesson in a Course might be extremely helpful.

**Comment by digitalchild** [comment]
Learn Pathways have replaced courses.
